OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -239,14 +241,18 @@ protected override void SetVar(string varName, DreamValue value) {
             case "hub_password":
             case "mob":
             case "name":
-            case "sleep_offline":
             case "status":
             case "version":
             case "visibility":
                 // Set it in the var dictionary, so reading at least gives the same value
                 base.SetVar(varName, value);
                 break;
 
+            case "sleep_offline":
+                base.SetVar(varName, value); // Only 1 and not-1 matter, but set it in case any user code cares what the actual value is
```

Comment:
BYOND is actually always giving 0 when trying to get `world.sleep_offline`

![image](https://github.com/user-attachments/assets/3ec8f6a1-3818-4fad-9c6b-852bc27a350b)
![image](https://github.com/user-attachments/assets/616940ab-a4b9-4f1f-a011-f8289661f6c0)

Reply:
Turns out BYOND is erroneously giving the opposite value (`0` for on and `1` for off). We should return the correct 0/1 value here.